From 2de5d31789aeaf59f7689429a61c2a9ac84e6f7f Mon Sep 17 00:00:00 2001
From: Rui Sousa <rui.sousa@nxp.com>
Date: Tue, 30 Nov 2021 15:36:07 +0100
Subject: [PATCH 11/48] HRPN-35 configs: Guest virtual console

Allow guest cells to use hypervisor calls for accessing hypervisor
virtual console.
This provides a very simple and cheap debug interface for guest low
level code.

Signed-off-by: Rui Sousa <rui.sousa@nxp.com>
---
 configs/arm64/cell-create.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/arm64/cell-create.h b/configs/arm64/cell-create.h
index 871784fc..20bcff11 100644
--- a/configs/arm64/cell-create.h
+++ b/configs/arm64/cell-create.h
@@ -23,7 +23,7 @@ struct {
 		.signature = JAILHOUSE_CELL_DESC_SIGNATURE,
 		.revision = JAILHOUSE_CONFIG_REVISION,
 		.name = CONFIG_INMATE_NAME,
-		.flags = JAILHOUSE_CELL_PASSIVE_COMMREG,
+		.flags = JAILHOUSE_CELL_PASSIVE_COMMREG | JAILHOUSE_CELL_VIRTUAL_CONSOLE_ACTIVE,
 
 		.cpu_set_size = sizeof(config.cpus),
 		.num_memory_regions = ARRAY_SIZE(config.mem_regions),
-- 
2.34.1

